content/docs/faq.md

```diff
@@ -305,6 +311,12 @@ purposes. However, it didn't receive much uptake as a development tool and had
 numerous issues with its design. In the end, we decided to remove it and split
 it out as a plugin.
 
+For a similar experience to `helm serve`, have a look at the local filesystem
```

Comment:
I just noticed this was also in #453. So a rebase might be needed once one of these PRs is merged

Reply:
oops, I thought I'd fixed that before submitting the PR. Everything seems to be OK though?